src/tasks/level10.ts

```diff
@@ -115,19 +115,20 @@ export const GiantQuest: Quest = {
         have($item`Plastic Wrap Immateria`) ? 25 : have($item`Gauze Immateria`) ? 20 : 15, // After that, just look for noncombats
       outfit: () => {
         if (forceItemPossible())
-          return { modifier: "-combat", avoid: $items`Kramco Sausage-o-Matic™` };
+          return { modifier: "-combat", avoid: $items`Kramco Sausage-o-Matic™`, equip: $items`bat wings` };
         else
           return {
             modifier: "-combat, item",
             avoid: $items`broken champagne bottle`,
+            equip: $items`bat wings`
           };
       },
       combat: new CombatStrategy()
         .macro(() => {
           if (have($item`Mohawk wig`)) return new Macro();
           if (haveEquipped($item`Fourth of May Cosplay Saber`) && get("_saberForceUses") < 5)
             return Macro.skill($skill`Use the Force`);
-          if (have($skill`Emotionally Chipped`) && get("_feelEnvyUsed") < 3)
+          if (have($skill`Emotionally Chipped`) && get("_feelEnvyUsed") < 3 && have($item`amulet of extreme plot significance`))
```

Comment:
Why wait until after the amulet? I would say you should just use envy on either ASAP.

Reply:
I can do that, but I'll need to add the amulet NC to the choice adventure script. I prioritized the amulet because healer had custom prioritization before I started touching this code.

Further, if you want to do that, I'd like to simply this into one airship step.

Reply:
Did you get to the airship with only 1 envy left? If you have 2 left it shouldn't matter which order you do them in.